libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Model/EventType.cs

```diff
@@ -19,7 +20,9 @@ public class EventType
 
         public string Ts { get; set; }
 
-        public string Item { get; set; }
+#pragma warning disable CA2227 // Collection properties should be read only
```

Comment:
Can we workaround this exclusion by setting the property as:

```
public JObject Item { get; } = new JObject();
```

Also, this looks like a breaking change, if it is necesary to fix a bug then I think is OK.

Reply:
Changed.